sslcontext-kickstart-bom/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.github.hakky54</groupId>
+        <artifactId>sslcontext-kickstart-parent</artifactId>
+        <version>7.4.10-SNAPSHOT</version>
+    </parent>
+    <artifactId>sslcontext-kickstart-bom</artifactId>
+    <name>sslcontext-kickstart-bom</name>
```

Comment:
I would prefer the name slighty different, what do think of `SSLContext Kickstart (Bill of Materials)` and also for description?